### PR TITLE
use ec2 runner for kube acceptance tests

### PR DESCRIPTION
**Pull request by jrhizor**
also rearranges the order to read a little cleaner on Github

### CODE REVIEW

**Comment by jrhizor** [comment]
@davinchia should actually be ready for review now

Any idea why `HOME`/`USER` aren't set up the same way on the EC2 runner as they are on the Github one? I made a pr `https://github.com/manusa/actions-setup-minikube/pull/46` to make this slightly better but it still requires the manual steps before and after the main minikube setup portion.

**Review by davinchia — APPROVED** [review]
Not completely sure, but the image we are running on our ec2 instances is an image I created. https://us-west-2.console.aws.amazon.com/ec2/v2/home?region=us-west-2#Images:sort=name

We could better mirror the proper github runner set up onto that image? Maybe a low priority ticket? What you have here looks good though.

**Comment by jrhizor** [comment]
@davinchia can you do a final once-over? going to merge since the last build failure was a transient one and the stopping worked.

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -294,18 +323,62 @@ jobs:
...
+      - name: Fix EC-2 Runner
+        run: |
+          mkdir -p /home/runner
+
       - name: Setup Minikube
         uses: manusa/actions-setup-minikube@v2.4.1
         with:
           minikube version: 'v1.21.0-beta.0'
           kubernetes version: 'v1.20.7'
+        # In case of elf-hosted EC2 errors, remove this env block.
```
Comment (davinchia) [inline .github/workflows/gradle.yml:335]: *self-hosted

**Pull request merged**